src/main/java/seedu/sellsavvy/logic/commands/ordercommands/EditOrderCommand.java

```diff
@@ -82,7 +82,7 @@ public CommandResult execute(Model model) throws CommandException {
         Order editedOrder = createEditedOrder(orderToEdit, editOrderDescriptor);
 
         OrderList orderList = model.getSelectedOrderList();
-        String feedbackToUser = !orderToEdit.isSameOrder(editedOrder)
+        String feedbackToUser = !orderToEdit.equals(editedOrder) // ensure that some field(s) is changed
```

Comment:
should i put equal or just delete this lol?

Reply:
not necessarily if we crazy enough we can do edit with the field exactly the same as before

Reply:
With equal around:
![image](https://github.com/user-attachments/assets/96ff3e42-7e80-4e8f-a6e0-2e7611ab37cf)
Without equal around 
![image](https://github.com/user-attachments/assets/82e4ed52-c841-43e7-a7b8-a35225169616)
(where the duplicate come from?)
Problems:
1) The problem about using equal is that since we never give warning for it, it is weird when there is another duplicate which already exist and we edit it nothing changed and it is same but no warning. 
2) We edit date to same thing and warning is given so not standardised.

Ideal solution:
Maybe give warning when edited fields are same as og? what u think

Reply:
personally thought it'll be a bit weird to give a warning even if the fields are the same, i don't remember seeing any app where if you edit a field and it's the same then they give you a warning.

Do you think we can create a `containsDuplicateOf(Order order)` method in `OrderList` which does the same thing as `contains(Order)` but also makes sure both orders are not the same as well?
e.g.
```
internalList.stream().anyMatch(order -> {
    if (toCheck.equals(order)) {
        return toCheck != order;
    }
    return false;
}
```
Then in `EditOrderCommand` 
```
        model.setOrder(orderToEdit, editedOrder);
        String feedbackToUser = orderList.containsDuplicateOf(editedOrder)
                ? ...
```

Reply:
return toCheck != order;
 won't work lol because edited order is not in the list unless we take in 2 parameters one og one edited

Reply:
done